src/main/java/org/bonitasoft/connectors/rest/AbstractRESTConnectorImpl.java

```diff
@@ -119,6 +136,31 @@ protected final List<List<?>> getUrlHeaders() {
         return headers;
     }
 
+    protected final Boolean getAddBonitaContextHeaders() {
+        final Boolean addContextHeaders = (Boolean) getInputParameter(ADD_BONITA_CONTEXT_HEADERS_INPUT_PARAMETER);
+        return addContextHeaders != null ? addContextHeaders : Boolean.FALSE;
```

Comment:
There is a `getInputParameter` with a default value, I guess it can be simplified by: 
```
return (Boolean) getInputParameter(ADD_BONITA_CONTEXT_HEADERS_INPUT_PARAMETER, Boolean.FALSE);
```